sbt/src/sbt-test/watch/watch-parser/test

```diff
@@ -0,0 +1,18 @@
+> ~; setStringValue foo; setStringValue bar
+
+> checkStringValue bar
+
+> ~;setStringValue foo;setStringValue bar; checkStringValue bar
```

Comment:
In this example, is the leading semicolon required, or optional?

Reply:
optional. I'll duplicate that test case without the leading `;`.